### PR TITLE
Ability to configure auto update

### PR DESCRIPTION
**Pull request by worksofliam**
* Adds new VS Code configuration to auto update the extension when connecting to systems.

### CODE REVIEW

**Review by edmundreinhardt — APPROVED** [review]
nice

**Review by edmundreinhardt — APPROVED** [review]
nice

File: src/connection/serverComponent.ts [hunk 1]
```diff
@@ -115,9 +120,24 @@ export class ServerComponent {
         const lastInstalledName = Config.getServerComponentName();
 
         if (lastInstalledName !== basename || this.installed === false) {
-          const updateQuestion = await window.showInformationMessage(`An update to the database server component is required: ${basename}`, `Update`);
+          let doUpdate = false;
```
Comment (ThePrez) [inline src/connection/serverComponent.ts:123]: Should this be something like?:
```javascript
let doUpdate = shouldAutoUpdate();
```
(current implementation may be preferred for readability with subsequent logic, so no assertions here)

**Review by ThePrez — APPROVED** [review]

**Comment by worksofliam** [comment]
Conflicts with #158

**Comment by worksofliam** [comment]
Server component does this by default now.

**Pull request closed without merging**